metrics/codahale.go

```diff
@@ -234,6 +237,23 @@ func (c *CodaHale) IncErrorsStreaming(routeId string) {
 	}
 }
 
+func (c *CodaHale) Close() {
+	close(c.quit)
```

Comment:
Maybe better to also wrap it into a `once.Do` as we do in other cases, because it will panic to call it twice.

Reply:
yes, but OTOH it would uncover incorrect closing.

Reply:
Could also return error if its already closed.